setup.py

```diff
@@ -23,10 +23,18 @@
           'console_scripts': [
               'ega-ingest = lega.ingest:main',
               'ega-verify = lega.verify:main',
-              'ega-keyserver = lega.keyserver:main',
               'ega-finalize = lega.finalize:main',
               'ega-conf = lega.conf.__main__:main',
           ]
       },
       platforms='any',
-      )
+      install_requires = [
+          'pika',
+          'aiohttp>=3.3.2',
+          'psycopg2>=2.8.4',
+          'PyYaml',
+          'boto3',
+          'requests',
+          #'crypt4gh', 'git+https://github.com/NBISweden/LocalEGA-cryptor.git'
```

Comment:
```suggestion
          'crypt4gh @ git+https://github.com/EGA-archive/crypt4gh.git' 
```

Reply:
ahhhh.... that's how it's done? Thanks... I'll update it

Reply:
I changed and then reverted, apparently it does not work.

Reply:
Sees to work when I build:
```
Step 7/25 : RUN pip install /root/LocalEGA
 ---> Running in 2add6cc469e2
Processing /root/LocalEGA
Requirement already satisfied: pika in /usr/local/lib/python3.6/site-packages (from lega==1.1) (1.0.1)
Requirement already satisfied: aiohttp in /usr/local/lib/python3.6/site-packages (from lega==1.1) (3.3.2)
Requirement already satisfied: psycopg2>=2.8.4 in /usr/local/lib/python3.6/site-packages (from lega==1.1) (2.8.4)
Requirement already satisfied: PyYaml in /usr/local/lib/python3.6/site-packages (from lega==1.1) (5.1.2)
Requirement already satisfied: boto3 in /usr/local/lib/python3.6/site-packages (from lega==1.1) (1.10.27)
Requirement already satisfied: requests in /usr/local/lib/python3.6/site-packages (from lega==1.1) (2.21.0)
Requirement already satisfied: crypt4gh@ git+https://github.com/EGA-archive/crypt4gh.git from git+https://github.com/EGA-archive/crypt4gh.git in /usr/local/lib/python3.6/site-packages (from lega==1.1) (1.0)
Requirement already satisfied: idna-ssl>=1.0 in /usr/local/lib/python3.6/site-packages (from aiohttp->lega==1.1) (1.1.0)
Requirement already satisfied: async-timeout<4.0,>=3.0 in /usr/local/lib/python3.6/site-packages (from aiohttp->lega==1.1) (3.0.1)
...
```

Reply:
Tox builds in a separate environment and it was telling me ```Direct url requirement are not allowed for dependencies```

Reply:
not getting that error, only error I get is  `pg_config` executable not found, but that i know of

Reply:
tox -r to rebuild fixes it :)